lte/gateway/Vagrantfile

```diff
@@ -43,7 +43,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # 2. Any changes to the magma networking should be propagated to magma_deb
     magma.vm.network "private_network", ip: "192.168.60.142", nic_type: "82540EM"
     # iperf3 trfserver routable IP.
-    magma.vm.network "private_network", ip: "192.168.129.1", nic_type: "82540EM"
+    # Set to an internal network so that the traffic server connects to the magma VM
+    # instead of the vagrant host, both of which use the same IP (192.168.129.1)
+    magma.vm.network "private_network", ip: "192.168.129.1", nic_type: "82540EM", virtualbox__intnet: true
     magma.vm.network "private_network", ip: "3001::10", nic_type: "82540EM"
```

Comment:
the ipv6 network  should be "internal" as well. Otherwise we could see similar issues there.

Reply:
Done